tests/frontier/opcodes/test_calldataload.py

```diff
@@ -0,0 +1,124 @@
+"""
+abstract: Test `CALLDATALOAD`
+    Test the `CALLDATALOAD` opcodes.
+
+"""
+
+import pytest
+
+from ethereum_test_forks import Frontier, Homestead
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+
+
+@pytest.mark.parametrize(
+    "calldataload_offset,memory_preset_code,call_args_size,expected_calldataload_memory",
+    [
+        (
+            0x00,
+            Bytecode(Op.MSTORE8(offset=0x0, value=0x25) + Op.MSTORE8(offset=0x1, value=0x60)),
+            0x02,
+            0x2560000000000000000000000000000000000000000000000000000000000000,
+        ),
+        (
+            0x01,
+            Bytecode(
+                Op.MSTORE(
+                    offset=0x0,
+                    value=0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF,
+                )
+                + Op.MSTORE8(offset=0x20, value=0x23),
+            ),
+            0x21,
+            0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF23,
+        ),
+        (
+            0x05,
+            Bytecode(
+                Op.MSTORE(
+                    offset=0x0,
+                    value=0x123456789ABCDEF0000000000000000000000000000000000000000000000000,
+                )
+                + Op.MSTORE8(offset=0x20, value=0x0)
+                + Op.MSTORE8(offset=0x21, value=0x24),
+            ),
+            0x22,
+            0xBCDEF00000000000000000000000000000000000000000000000000024000000,
+        ),
+    ],
+    ids=["two_bytes", "word_n_byte", "34_bytes"],
+)
+@pytest.mark.valid_from("Frontier")
+def test_calldataload(
+    state_test: StateTestFiller,
+    fork: str,
+    calldataload_offset: int,
+    memory_preset_code: Bytecode,
+    call_args_size: int,
+    expected_calldataload_memory: Account,
```

Comment:
```suggestion
    expected_calldataload_memory: int,
```
Correct type seems to be `int`, based on the values in your `pytest.mark.parametrize`.